soql-stdlib/src/main/scala/com/socrata/soql/functions/SoQLFunctions.scala

```diff
@@ -166,6 +166,15 @@ object SoQLFunctions {
   val Overlaps = f("overlaps", FunctionName("overlaps"), Map("a" -> GeospatialLike, "b" -> GeospatialLike),
     Seq(VariableType("a"), VariableType("b")), Seq.empty, FixedType(SoQLBoolean))(
     "Return the rows that where the locations 'spatially overlap', meaning they intersect, but one does not completely contain another and they share interior points")
+  
+  val Intersection = f("intersection", FunctionName("intersection"), Map("a" -> GeospatialLike, "b" -> GeospatialLike),
+    Seq(VariableType("a"), VariableType("b")), Seq.empty, VariableType("a"))(
```

Comment:
This says that an intersection can be taken between any two geometries and will have the same type as the first one.  Is that true?  It seems unlikely.  In general, it seems unlikely that the type of the resulting geometry can be tied to to the type of the input geometries in any particular way (e.g., the intersection of two polygons may be a multipolygon)

Reply:
(also, is this useful on non-polygon-like geometry types?)

Reply:
fair point - I'm going to rename this function such that the "this is between two polygons" and results in a single multipolygon (or null) is clear